api/scpca_portal/metadata_file.py

```diff
@@ -96,6 +96,8 @@ def write_metadata_dicts(list_of_dicts: List[Dict], output_file_path: str, **kwa
         "fieldnames", utils.get_sorted_field_names(utils.get_keys_from_dicts(list_of_dicts))
     )
     kwargs["delimiter"] = kwargs.get("delimiter", common.TAB)
+    # Add any missing fields with their value "NA"
```

Comment:
```suggestion
    # By default fill missing values with "NA"
```

Reply:
Updated at f2cdbe70522176e6b7968dd2e9d74c302fccc41b, thank you!